src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -1012,6 +1012,19 @@ def prepare_create(self, daemon_spec: CephadmDaemonDeploySpec) -> CephadmDaemonD
         return daemon_spec
 
 
+class CephExporterService(CephService):
+    TYPE = 'ceph-exporter'
+
+    def prepare_create(self, daemon_spec: CephadmDaemonDeploySpec) -> CephadmDaemonDeploySpec:
+        assert self.TYPE == daemon_spec.daemon_type
+        daemon_spec.final_config, daemon_spec.deps = self.generate_config(daemon_spec)
+        return daemon_spec
+
+    def generate_config(self, daemon_spec: CephadmDaemonDeploySpec) -> Tuple[Dict[str, Any], List[str]]:
+        assert self.TYPE == daemon_spec.daemon_type
+        return {}, []
```

Comment:
these don't need any config at all? They don't need any info on where other daemons are or anything?

Reply:
The configuration will be introduced in the next iteration. port, addrs, cert,key,etc.. will be the config we're planning to introduce for this daemon